Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -503,6 +495,12 @@ class BrowserViewController: UIViewController {
         self.presentViewController(alert, animated: true, completion: nil)
     }
 
+    private func shouldRestoreTabs() -> Bool {
+        guard let tabsToRestore = tabManager.tabsToRestore() else { return false }
+        let onlyNoHistoryTabs = !tabsToRestore.every { $0.sessionData?.urls.count > 1 }
```

Comment:
Hm...separate bug, but this test isn't completely reliable: any time we open a new tab from long-pressing a link, that tab will only have a single back/forward item (the opened page), and it's not the home page. Probably not a high priority bug since this only happens a) when showing the crash prompt, and b) only if *all* of the opened tabs have no back/forward history.

That said, I think it'd be an easy fix if you changed this to:
`!tabsToRestore.every { $0.sessionData?.urls.count > 1 || !AboutUtils.isAboutHomeURL($0.sessionData?.urls.first) }`